db/seeds.rb

```diff
@@ -1,33 +1,40 @@
-Rails.logger.info "1/9 Creating Organizations"
+Rails.logger.info "1/10 Creating Organizations"
 
-3.times do
+40.times do
+  hex = SecureRandom.hex 7
   Organization.create!(
     name: Faker::SiliconValley.company,
     summary: Faker::Company.bs,
     remote_profile_image_url: logo = Faker::Company.logo,
     nav_image: logo,
     url: Faker::Internet.url,
-    slug: "org#{rand(10_000)}",
-    github_username: "org#{rand(10_000)}",
-    twitter_username: "org#{rand(10_000)}",
+    slug: "org#{hex}",
+    github_username: "org#{hex}",
+    twitter_username: "org#{hex}",
     bg_color_hex: Faker::Color.hex_color,
-    text_color_hex: Faker::Color.hex_color,
+    text_color_hex: Faker::Color.hex_color
   )
 end
 
 ##############################################################################
 
-Rails.logger.info "2/9 Creating Users"
+Rails.logger.info "2/10 Creating Users"
 
 roles = %i[level_1_member level_2_member level_3_member level_4_member
            workshop_pass]
 User.clear_index!
-10.times do |i|
+
+
+
+100.times do |i|
+  name = Faker::Name.unique.name
+  puts "username #{Faker::Internet.username(name)}"
```

Comment:
I wouldn't display 100 (or a 1000 in case of the user's multiplication factor) user names on the console, what's to gain?

Reply:
If you are testing out pagination, you would.

Reply:
@omenking, @rhymes was not questioning inserting that number of records in the DB, he was just pointing out that the stdout output during seeding would be excessive.